statsmodels/tsa/automatic/exponentialsmoothing.py

```diff
@@ -0,0 +1,61 @@
+"""Automatic selection of the Exponential Smoothing Model."""
+import numpy as np
+import warnings
+import statsmodels.api as sm
+
+# from statsmodels.tsa.automatic import transform
+
+
+def auto_es(endog, measure='aic', seasonal_periods=1, damped=False,
+            lambda_val=None, additive_only=False, alpha=None, beta=None,
+            gamma=None, phi=None):
+    if damped:
+        trends = ["add", "mul"]
+    else:
+        trends = ["add", "mul", None]
+    # damped
+    seasonal = ["add", "mul", None]
+    # TODO use lambda in fitting ES models
+    # if lambda_val == 'auto':
+    #     lambda_val = transform.predict_lambda(endog)
+    # else:
+    #     if lambda_val is not None:
+    #         additive_only = True
+    if additive_only:
+        trends = ["add"]
+        seasonal = ["add"]
+    min_aic = np.inf
```

Comment:
Probably want this to be `min_measure` or something, now that it's not only aic.

Reply:
Thanks for pointing out. I'll do that.